### DISCUSSION TITLE
Advice 📚🚀 Best libraries to power projects with Next.js? 🤔

### DISCUSSION THREAD

**Discussion by Miguel-ectil** [community/community#79099] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hey guys! 👋🏿 My name is Miguel Ectil, I'm a junior Front-End developer and I'm looking for advice from more experienced developers here in the community. I am currently immersed in a project with Next.js and I would like to know which libraries you recommend or have used successfully in this context. Which ones can improve the development experience or offer useful features? I would be very grateful if you could share why you consider these libraries valuable. Thank you in advance for sharing your experiences and knowledge! 🌟

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Miguel-ectil , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
